### PR TITLE
Add Accepted Talks

### PR DESCRIPTION
**Pull request by anah**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -105,6 +105,33 @@ <h5 class="sponsor-heading">sponsored by</h5>
             <img src="assets/images/speakers/ian_ozsvald.jpg" class="img-responsive img-circle" alt="Ian Ozsvald" width="200">
           </div>
           <div class="col-sm-12">
+            <h2>Accepted Talks</h2>
```
Comment (frvi) [inline index.html]: Maybe use another title here, like just "Talks", or "Speakers"?

Comment (nailor) [inline index.html]: :+1: on that

**Comment by anah** [comment]
I will merge this tomorrow morning. I've scheduled a Buffer post for 9 am.

**Comment by nailor** [comment]
:+1:

**Pull request merged**